tests/functional/object/mcg/test_deny_http.py

```
@@ -0,0 +1,336 @@
+        """
+        test_data = "deny-http-test-data"
+        s3_client.put_object(Bucket=bucket_name, Key=object_key, Body=test_data)
+        response = s3_client.get_object(Bucket=bucket_name, Key=object_key)
+        retrieved_data = response["Body"].read().decode()
+        assert (
+            retrieved_data == test_data
+        ), f"Data mismatch: expected '{test_data}', got '{retrieved_data}'"
+        return True
+
+    @staticmethod
+    def _verify_curl_access(pod, url, should_succeed=True):
```

Ruff (0.15.14)

[warning] Missing return type annotation for staticmethod `_verify_curl_access`

Add return type annotation: `bool | None`

(ANN205)